themes/default/content/docs/concepts/inputs-outputs/lifting.md

```diff
@@ -0,0 +1,316 @@
+---
+title_tag: "Accessing Nested Properties with Lifting | Inputs and Outputs"
+meta_desc: "Learn how to access nested output values using the lifting method in Pulumi."
+title: Accessing nested properties with Lifting
+h1: Accessing nested properties with Lifting
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  concepts:
+    weight: 4
+    parent: inputs-outputs
+---
+
+While often, Outputs return asynchronous values that wrap primitive types like strings or integers, sometimes an output has an object with deeply nested values. These properties need to be passed to other inputs as well.
```

Comment:
This wording feels awkward. Maybe it's just the extra comma after "While often"?

Reply:
The lifting page will be updated in a future phase of edits. This round is just the reorg, apply, and all pages for now